src/services/submit.ts

```diff
@@ -196,10 +217,7 @@ export default async function submit({
       submissionId: data.submissionId,
       isUploadingAttachments: false,
       downloadSubmissionPdfUrl: downloadSubmissionPdfUrl(),
-      getDownloadSubmissionPdfUrl: !data.pdfAccessToken
-        ? undefined
-        : (configurationId: string) =>
-            `${tenants.current.apiOrigin}/forms/${formSubmission.definition.id}/submissions/${data.submissionId}/pdf-document?accessToken=${data.pdfAccessToken}&configurationId=${configurationId}`,
+      downloadSubmissionPdfs: downloadSubmissionPdfs(),
```

Comment:
Seems ~~a lot of~~ some duplicate logic between the two properties. Two proposals:

1. breaking change and only return the new `downloadSubmissionPdfs` property. If `allowPDFDownload` is not an array, still return an array with one item
2. Revert back to only returning `downloadSubmissionPdfUrl` and append the `configurationId` query string parameter to the url in `apps-react`.

I vote 2.

Reply:
3. Revert back to only returning downloadSubmissionPdfUrl and export a new function from `apps` that takes in `FormSubmissionResult` and `configurationId` and gives you back an absolute URL with the configurationId query string parameter included.

Reply:
I'd thought about 3 also, for what it's worth. 2 seems the least amount of work.

Reply:
While maintaining a rift between apps and apps-react, I don't like adding more assumed knowledge to apps-react.